notebooks/225-distilbert-sequence-classification/225-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,319 @@
+    "                  'attention_mask': {0: 'batch_size', 1: 'sequence'},\n",
+    "                  'logits': {0: 'batch_size', 1: 'sequence'}},\n",
+    "    do_constant_folding=True,\n",
+    "    opset_version=13,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fff79fc",
+   "metadata": {},
+   "source": [
+    "# Model Optimizer\n",
+    "\n",
+    "[Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0f48c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer_command = f\"mo \\\n",
+    "    --input_model {onnx_model} \\\n",
+    "    --output_dir {MODEL_DIR} \\\n",
+    "    --model_name {checkpoint} \\\n",
+    "    --input input_ids,attention_mask \\\n",
+    "    --input_shape [1,128],[1,128]\"\n",
+    "! $optimizer_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "OpenVINO™ Runtime uses [Infer Request](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Infer_request.html) mechanism which allows running models on different devices in asynchronous or synchronous manners. The model graph is sent as an argument to the OpenVINO API and an inference request is created. The default inference mode is AUTO but it can be changed according to requirement and hardwares available. You can explore the different inference modes and their usage [here.](https://docs.openvino.ai/latest/openvino_docs_Runtime_Inference_Modes_Overview.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e31a2644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.filterwarnings(\"ignore\")\n",
+    "core = ov.Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR) / checkpoint).with_suffix(\".xml\"))\n",
+    "compiled_model = core.compile_model(ir_model_xml)\n",
+    "infer_request = compiled_model.create_infer_request()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "de01fccc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "Defining a softmax function to extract\n",
+    "the prediction from the output of the IR format\n",
+    "\"\"\"\n",
+    "\n",
+    "\n",
+    "def softmax(x):\n",
+    "    e_x = np.exp(x - np.max(x))\n",
+    "    return e_x / e_x.sum()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e778507",
+   "metadata": {},
+   "source": [
+    "# Inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cc0c91a6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
```

Comment:
please move doc strings inside function. It should looks like:
```python
def infer(input_text):
   """ 
   Some description
   Params:
   Returns:
  """
  <your code>
```

Reply:
Done